### ISSUE TITLE
Missing score from pylint

### ISSUE THREAD

**Issue by dsladek** [AtakamaLLC/lint-diffs#14]
Hi there would be nice to adds some variable to force printing score from pylint.  What I often observed is that I'm getting 
for example 
```python
 === pylint: mine=5, always=0
 The following messages were raised:
   - warning message issued
   - refactor message issued
   - convention message issued
 No fatal messages detected.  Exiting gracefully...
````

but score result is missing.  When I add --debug attribute, score is visible as commented, but lot of redundant lines are printed.

same code with --debug
```python
# 
 # -----------------------------------
 # Your code has been rated at 9.11/10
 # 
 # 
 === pylint: mine=5, always=0
 The following messages were raised:
   - warning message issued
   - refactor message issued
   - convention message issued
 No fatal messages detected.  Exiting gracefully..
```

**Comment by earonesty** [comment]
The score, however, isn't quite relevant, since its a score based on all lines of the changed files, not based on diffs only.   The only good way to make a score, is to figure out pylint's algorithm and reproduce it.

Which I'm fine with if anyone happens to know it.

**Comment by earonesty** [comment]
this will be pretty hard to calculate in a way that's not pylint-specific: `10.0 - ((float(5 * error + warning + refactor + convention) / statement) * 10)`.   

 - generic mechanism for mapping lines to classified categories (regex for error vs warning vs convention)
 - generic mechanism for inputting a score formula (using those category names as vars)
 - mechanism for storing the lines that factor into a "previous known scoreset"
 - given a "previous known scoreset", compute a relative diff-score that can be applied
 - now you can produce the new "scoreset" (given the old), and store it, and it should track diffs over time
 
 a "scoreset" is the set of all categorized linter errors along with their line numbers, sufficient to calculate the score
